pyapp/{{ cookiecutter.slug }}/{{ cookiecutter.package_name }}/app.py

```diff
@@ -0,0 +1,9 @@
+import time
+
+from {{ cookiecutter.package_name }}.core import hello_world
+
+
+def run():
+    while True:
+        print(hello_world())
+        time.sleep(1)
```

Comment:
This is just some totally arbitrary code in order to provide a continuously-running command for `make dev` and `make docker-run` to call. Obviously the expectation is that projects will change this file